### PR TITLE
Add CAAM support for 8DXL platforms

### PR DESCRIPTION
**Pull request by clementfaure**

### CODE REVIEW

**Comment by clementfaure** [comment]
Thanks @jforissier 
Could this PR be included in the 4.0,0 release?

**Comment by jforissier** [comment]
> Thanks @jforissier Could this PR be included in the 4.0,0 release?

Absolutely! The `4.0.0-rc1` tag is not created yet (I will make it on Friday), so there is no restriction on what can be merged at this point.

**Comment by jenswi-linaro** [comment]
`Acked-by: Jens Wiklander <jens.wiklander@linaro.org>`

File: core/arch/arm/plat-imx/conf.mk [hunk 1]
```diff
@@ -401,6 +401,9 @@ endif
 ifneq (,$(filter $(PLATFORM_FLAVOR),mx8dxlevk))
 CFG_DDR_SIZE ?= 0x40000000
 CFG_UART_BASE ?= UART0_BASE
+CFG_NSEC_DDR_1_BASE ?= 0x800000000UL
+CFG_NSEC_DDR_1_SIZE  ?= 0x400000000UL
```
Comment (etienne-lms) [inline core/arch/arm/plat-imx/conf.mk:405]: 2 space chars

**Review by etienne-lms — COMMENTED** [review]
`Acked-by: Etienne Carriere <etienne.carriere@foss.st.com>`

File: core/arch/arm/plat-imx/conf.mk (hunk 1 above)
Comment (clementfaure) [reply core/arch/arm/plat-imx/conf.mk]: Fixed

**Pull request merged**